backend/extractor/hdqmextract.py

```diff
@@ -24,7 +24,8 @@
 from helpers import batch_iterable, exec_transaction, get_all_me_names
 
 CFGFILES = 'cfg/*/*.ini'
-ROOTFILES = '/eos/cms/store/group/comm_dqm/DQMGUI_data/*/*/*/DQM*.root'
+#ROOTFILES = '/eos/cms/store/group/comm_dqm/DQMGUI_data/*/*/*/DQM*.root'
+ROOTFILES = '/afs/cern.ch/user/s/seungjun/public/koco/*/*/*/DQM*.root'
```

Comment:
I believe this line may cause trouble for the rest of subsystems. We may copy the GEM root files to the central eos DQM folder instead.
What do you think @andrius-k  ?

Reply:
i am sorry that is my mistake..
i fix it